### PR TITLE
add homework1

### PR DESCRIPTION
**Pull request by FarzanehAhmadi**

### CODE REVIEW

File: html-css/week1/Homework 1/index.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <nav id="desctop-nav">
```
Comment (Hadis-jamali) [inline html-css/week1/Homework 1/index.html:10]: Wrap nav in a header tag for semantic clarity

File: html-css/week1/Homework 1/index.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            
+            <!-- GitHub Info -->
+            <div class="contact-info">
+              <img src="assets/github.png" alt="GitHub Icon" class="icon">
+              <p>GitHub: <a href="https://github.com/FarzanehAhmadi" target="_blank">FarzanehAhmadi</a></p>
+            </div>
+          </div>
+        </div>
+      </section>
+    <!-- Footer Section -->
```
Comment (Hadis-jamali) [inline html-css/week1/Homework 1/index.html:166]: remove all comments

File: html-css/week1/Homework 1/styles.css [hunk 3]
```diff
@@ -0,0 +1,540 @@
+/*General*/
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@300;400;500;600&display=swap');
```
Comment (Hadis-jamali) [inline html-css/week1/Homework 1/styles.css:2]: Separate @import for readability then Add a font-family 'poppins'

File: html-css/week1/Homework 1/styles.css [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+    color: rgb(96, 38, 84);
+   
+    text-decoration:underline;
+    text-decoration-color: rgb(94, 54, 95);
+    transition: all 400ms ease;
+}
+/*Section*/
+#home{
+    
+    margin: 0 10rem;
```
Comment (Hadis-jamali) [inline html-css/week1/Homework 1/styles.css:63]: you have 2 margin and also you used px and rem together.  remove second margin and merge it to the first one with rem.

File: html-css/week1/Homework 1/styles.css [hunk 5]
```diff
@@ -0,0 +1,540 @@
...
+    }
+  
+    .footer-section.map iframe {
+      width: 100%;
+      height: 150px;
+    }
+  }
+  
+  /* Tablet (600px to 768px) */
+  @media (min-width: 601px) and (max-width: 768px) {
```
Comment (Hadis-jamali) [inline html-css/week1/Homework 1/styles.css:478]: you can add just one media query with max-width: 600px

**Review by Hadis-jamali — COMMENTED** [review]
well done Farzaneh✨

File: html-css/week1/Homework 1/index.html (hunk 2 above)
Comment (FarzanehAhmadi) [reply html-css/week1/Homework 1/index.html:166]: Should I remove comments from the CSS as well? I added them for readability.

**Pull request merged**